### ISSUE TITLE
Content / Style Tradeoff. What are the weight values?

### ISSUE THREAD

**Issue by fastlater** [cysmith/neural-style-tf#45]
With respect to the content/ style tradeoff, 
What are the weight values used in the example Red Canna + Lion shown in the Readme?
I would like to replicate the results. Any changes in other parameters different than style weight?

Someone who experimented with the code, could give me some hints about the effect of changing the tv weight, temporal weight and content  loss function?
I am losing too much of the content even when I am only using style weight = 10.

**Comment by ProGamerGov** [comment]
The total variance weight essentially a blurring effect. Generally I keep it at 0 on [Neural-Style](https://github.com/jcjohnson/neural-style), and that should apply to neural-style-tf as well. 

What image size are you using? Image size plays a large role in the level of stylization in the final output. 

Generally the closer the image size is to that of the model's training data, the more stylization that occurs, and the more radical the changes are. Larger image sizes are the opposite of this. To counteract this, the multiscale resolution technique is used, where you first generate the image at a small image size, and then take that output and run it through again with the same settings but a higher image size. This process is repeated until you reach your desired output size.

There's an entire wiki section for the Lua implementation of the algorithm which is applicable to neural-style-tf: 

https://github.com/jcjohnson/neural-style/wiki

https://github.com/jcjohnson/neural-style/wiki/For-Beginners

You can also check out: https://github.com/manuelruder/artistic-videos for temporal weight information in relation to videos/gifs.

**Comment by cysmith** [comment]
If you look in the examples folder of the Lion + Red Canna, the style weight is the value in the image name (e.g. output_1000000.png has style weight 1e6). Temporal weight should only be if you're doing video.
